README.md

```diff
@@ -28,6 +28,7 @@ reasons for its fast speed, refer to [benchmarks](./README.md#Benchmarks) sectio
 storage services and local filesystems.
 
 - List buckets and objects
+- Check if objects and buckets exist
```

Comment:
I think the sentence above already covers this point. Could you remove the line and add a subsection under the 'usage' section to demonstrate how to write and read metadata of an object?

Reply:
👍